### PR TITLE
ARM64 compatibility

### PR DESCRIPTION
**Pull request by lmcjt37**
- Address deprecated ViewPropTypes in JS, allowing us to remove the patch entirely
- Update SDKs for ARM64 compatibility and fix updated ad constants

### CODE REVIEW

File: ios/RNAdManageNativeView.m [hunk 1]
```diff
@@ -170,13 +170,13 @@ - (void)reloadAd {
...
-            CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
-            [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
-        }
+        // NSDictionary *location = [_targeting objectForKey:@"location"];
+        // if (location != nil) {
+        //     CGFloat latitude = [[location objectForKey:@"latitude"] doubleValue];
+        //     CGFloat longitude = [[location objectForKey:@"longitude"] doubleValue];
+        //     CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
+        //     [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
+        // }
```
Comment (amrfarid140) [inline ios/RNAdManageNativeView.m:179]: Remove?

**Review by amrfarid140 — APPROVED** [review]
LGTM

File: ios/RNAdManageNativeView.m (hunk 1 above)
Comment (lmcjt37) [inline ios/RNAdManageNativeView.m:179]:
```suggestion
```

File: ios/RNAdManagerBannerView.m [hunk 2]
```diff
@@ -123,13 +123,13 @@ - (void)createViewIfCan
...
-            CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
-            [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
-        }
+        // NSDictionary *location = [_targeting objectForKey:@"location"];
+        // if (location != nil) {
+        //     CGFloat latitude = [[location objectForKey:@"latitude"] doubleValue];
+        //     CGFloat longitude = [[location objectForKey:@"longitude"] doubleValue];
+        //     CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
+        //     [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
+        // }
```
Comment (lmcjt37) [inline ios/RNAdManagerBannerView.m:132]:
```suggestion
```

File: ios/RNAdManagerInterstitial.m [hunk 3]
```diff
@@ -94,13 +94,13 @@ + (BOOL)requiresMainQueueSetup
...
-                CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
-                [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
-            }
+            // NSDictionary *location = [_targeting objectForKey:@"location"];
+            // if (location != nil) {
+            //     CGFloat latitude = [[location objectForKey:@"latitude"] doubleValue];
+            //     CGFloat longitude = [[location objectForKey:@"longitude"] doubleValue];
+            //     CGFloat accuracy = [[location objectForKey:@"accuracy"] doubleValue];
+            //     [request setLocationWithLatitude:latitude longitude:longitude accuracy:accuracy];
+            // }
```
Comment (lmcjt37) [inline ios/RNAdManagerInterstitial.m:103]:
```suggestion
```

**Pull request merged**